### PR TITLE
Adding an alert at the tap of the button to display a message as appropriate

### PR DESCRIPTION
**Pull request by YvesC88**
It is a small pull request to check if my practices are good to display a message to the user as the case may be when using the nearbySearch button

### CODE REVIEW

File: MonMaraicher/Features/Map/MapViewModel.swift [hunk 1]
```diff
@@ -13,6 +13,8 @@ final class MapViewModel: ObservableObject {
     @Published var selectedFarmer: Farmer?
     @Published var allFarmers: [Farmer] = []
     @Published var mapCameraPosition: MapCameraPosition = .userLocation(fallback: .automatic)
+    @Published var isErrorSearchFarmerPresented = false
+    @Published var titleError = ""
```
Comment (MathisDetourbet) [inline MonMaraicher/Features/Map/MapViewModel.swift:17]: errorTitle

File: MonMaraicher/Features/Map/MapView.swift [hunk 2]
```diff
@@ -54,6 +54,8 @@ extension MapView {
                 .background(Circle()
                     .fill(.thinMaterial))
         }
+        .alert(viewModel.titleError, isPresented: $viewModel.isErrorSearchFarmerPresented, actions: {
```
Comment (MathisDetourbet) [inline MonMaraicher/Features/Map/MapView.swift:57]: Why leave action empty?

**Review by MathisDetourbet — APPROVED** [review]

File: MonMaraicher/Features/Map/MapViewModel.swift (hunk 1 above)
Comment (YvesC88) [reply MonMaraicher/Features/Map/MapViewModel.swift:17]: Oh my god. It’s my French side

File: MonMaraicher/Features/Map/MapView.swift (hunk 2 above)
Comment (YvesC88) [reply MonMaraicher/Features/Map/MapView.swift:57]: I could have put a button to manage each alert message.

Change the distance if no farmer found and go to the settings if the user did not accept the location but it took a little more time to create these methods.

I’m going to create them soon

File: MonMaraicher/Features/Map/MapViewModel.swift (hunk 1 above)
Comment (MathisDetourbet) [reply MonMaraicher/Features/Map/MapViewModel.swift:17]: Same above: isSearchFarmerErrorPresented

File: MonMaraicher/Features/Map/MapView.swift (hunk 2 above)
Comment (MathisDetourbet) [reply MonMaraicher/Features/Map/MapView.swift:57]: Add a todo then, easier for me to understand ;)

**Pull request merged**